src/status_im/transactions/views/list_item.cljs

```diff
@@ -30,10 +30,11 @@
 
 (defview view [{:keys [to value id] :as transaction} on-deny]
   [recipient [:contact-by-address to]]
-  (let [eth-value      (.fromWei js/Web3.prototype value "ether")
-        value          (str (i18n/label-number eth-value) " ETH")
+  (let [safe-value     (* (Math/round (/ value 10000)) 10000)
```

Comment:
Comment from other PR by @rasom:

We can't use this because
```
(let [value 11111122223333441234]
  (= value (* (Math/round (/ value 10000)) 10000)))
=> false
```